src/pixi.inspector.js

```diff
@@ -584,7 +584,10 @@
 					case game.SceneMain: return 'game.SceneMain';
 				}
 			}
-			return 'Unknown';
+			if(!node.constructor.name){
```

Comment:
yoda condition, also spacing is not consistent with the basecode, and ; missing 
the following will be fine:
if (!!node.constructor.name) {
    return node.constructor.name;
}
return 'Unknown';